### ISSUE TITLE
Feature request: "View only" mode

### ISSUE THREAD

**Issue by ericras** [unl/UNL_MediaHub#363]
"Then, my only hesitation utilizing MediaHub is that I’ve noticed no option to make videos "view only" for the user and remove the download link?  We may have some restrictions with licensed materials in the future that can only be view only and wondering if there is an internal workaround"

**Comment by tommyneu** [comment]
This idea came back up in conversations now. Might be worth looking into making videos not "downloadable". I think it would be best if it was an opt-out thing, where if you didn't want people downloading your video you could turn it off

**Comment by tommyneu** [comment]
Yeah it would be a soft "view only" mode, but it would stop the average person on mediahub
